src/try-convert/UpdateTryConvert.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Diagnostics;
+namespace MSBuild.Conversion
+{
+    public static class UpdateTryConvert
+    {
+        public static void Update()
+        {
+            using (var process = new Process())
+            {
+                process.StartInfo.UseShellExecute = true;
+                process.StartInfo.FileName = "dotnet";
+                process.StartInfo.Arguments = "tool update --global try-convert";
```

Comment:
@Johnmancini30 what is the tool is not installed as a global tool?

Reply:
Okay, responded